docs/starters.yml

```diff
@@ -3428,3 +3428,22 @@
     - Page Transitions
     - Blog Post Template
     - Sitemap & Robots.txt generation
+- url: https://advancedblog.netlify.com/
+  repo: https://github.com/Bvlktech/AdvancedBlog
+  description: Advanced Blog Starter for GatsbyJS
+  tags:
+    - CMS: NetlifyCMS
+    - Styling: CSS-in-JS
```

Comment:
Hi,

Both CMS and Styling tags do contain any whitespace between the `:` colon and the text. Another point to note, it is `Netlify` and not `NetlifyCMS`. Please, edit these two lines into:

```md
- CMS:Netlify
- Styling:CSS-in-JS
```

For more info, check the [Categories](https://www.gatsbyjs.org/starters/?v=2) section.

Reply:
Yes sorry i will fix this it was a mistake on my part